parlai/tasks/self_chat/build.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+def build(opt):
```

Comment:
I don't think you need this file at all.

Reply:
ah yea you prolly right